src/components/Members.jsx

```diff
@@ -30,16 +23,11 @@ function Members() {
           )}
         </div>
       </div>
-      <div className="members"> 
+      <div className="contributors"> 
         <h3>Meet the Contributors</h3>
-        <div className="row">
-          {members.map((member) =>
-            <div key={member.id}>
-              <img className='avatar member-avatar' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />
-              <h4>{member.name}</h4>
-            </div>
-          )}
-        </div>
+          <a className="contributors-link" href="https://github.com/FCCColumbus/cbus-web/graphs/contributors" rel="noreferrer" target="_blank">
+            <img className='members-avatar' src='https://contrib.rocks/image?repo=FCCColumbus/cbus-web' alt='beautiful headshots of our members' />         
```

Comment:
Suggest renaming this style class to `contributors-avatar`, since it applies only to Contributors and not Admins, who are also "Members".